### PR TITLE
UWaterloo Amandaegraham edit

### PR DESCRIPTION
**Pull request by amandaegraham**
Edited HTML, Interpolation and Template Expression Operator sections 

Changes: 
- Changed tense from past to present 
- Reworded sentences 
- Added acronym descriptions 
- Changed concatenates to links 
- Rewrote sentences to not start with 'and'
- Changed pip operator order to have introduction 
- Added ':' before example descriptions

### CODE REVIEW

File: public/docs/ts/latest/guide/template-syntax.jade [hunk 1]
```diff
@@ -1433,7 +1433,7 @@ block json-pipe
   ### The safe navigation operator ( ?. ) and null property paths
 
   The Angular **safe navigation operator (`?.`)** is a fluent and convenient way to guard against null and undefined values in property paths.
-  Here it is, protecting against a view render failure if the `currentHero` is null.
+  Here it is used to protect against against a view render failure if the `currentHero` is null:
```
Comment (bampakoa) [inline public/docs/ts/latest/guide/template-syntax.jade]: duplicate word **against**

File: public/docs/ts/latest/guide/template-syntax.jade [hunk 2]
```diff
@@ -1446,10 +1446,10 @@ block dart-safe-nav-op
 +makeExample('template-syntax/ts/app/app.component.html', 'safe-1')(format=".")
 :marked
   The view still renders but the displayed value is blank; we see only "The title is" with nothing after it.
-  That is reasonable behavior. At least the app doesn't crash.
+  That is reasonable a behavior. At least the app doesn't crash.
```
Comment (bampakoa) [inline public/docs/ts/latest/guide/template-syntax.jade]: I think this must be the other way round 

> reasonable a

File: public/docs/ts/latest/guide/template-syntax.jade (hunk 1 above)
Comment (kwalrath) [inline public/docs/ts/latest/guide/template-syntax.jade]: I'm a little confused. I believe I just saw this text in a commit that I (belatedly) commented on: https://github.com/angular/angular.io/commit/d0811714d54cda1882f8287e095bf182ce39421d#commitcomment-18338618

File: public/docs/ts/latest/guide/template-syntax.jade (hunk 2 above)
Comment (kwalrath) [inline public/docs/ts/latest/guide/template-syntax.jade]: Again, I commented on this in https://github.com/angular/angular.io/commit/d0811714d54cda1882f8287e095bf182ce39421d#commitcomment-18338618

File: public/docs/ts/latest/guide/template-syntax.jade [hunk 3]
```diff
@@ -1446,10 +1446,10 @@ block dart-safe-nav-op
...
 
-  Suppose the template expression involves a property path, as in this next example
-  where we’re displaying the `firstName` of a null hero.
+  Suppose the template expression involves a property path, as seen in this next example
```
Comment (kwalrath) [inline public/docs/ts/latest/guide/template-syntax.jade]: And here: https://github.com/angular/angular.io/commit/d0811714d54cda1882f8287e095bf182ce39421d#commitcomment-18338618

**Comment by kwalrath** [comment]
I should finish taking a look, but in the meantime, please see my comments on https://github.com/angular/angular.io/commit/d0811714d54cda1882f8287e095bf182ce39421d.

**Comment by kwalrath** [comment]
Closing this PR since we have no CLA.

**Pull request closed without merging**